app.js

```diff
@@ -25,4 +25,126 @@ for (let i = 0; i < options.length; i += 1) {
+  },
+
+  {
+    name: 'Multi-Post stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    imageLink: './Images/Snapshoot-page-3.png',
+    technologies: ['html', 'css', 'javascript'],
+    role: 'Back end dev',
+    company: 'Canopy',
+    year: '2015',
+    sourceLink: 'https://github.com/Christianib003/Portfolio-Moblie-Skeleton-Version',
+    demoLink: 'https://christianib003.github.io/Portfolio-Moblie-Skeleton-Version/',
+  },
+
+  {
+    name: 'Tonic',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    imageLink: './Images/Snapshoot-page-4.png',
+    technologies: ['html', 'css', 'javascript'],
+    role: 'Back end dev',
+    company: 'Canopy',
+    year: '2015',
+    sourceLink: 'https://github.com/Christianib003/Portfolio-Moblie-Skeleton-Version',
+    demoLink: 'https://christianib003.github.io/Portfolio-Moblie-Skeleton-Version/',
+  },
+];
+
+for (let i = 0; i < projectsList.length; i += 1) {
+  workSection.innerHTML += `
+    <div class="card">
+      <div class="media">
+          <img class="image" src="${projectsList[i].imageLink}" alt="${projectsList[i].name}">
+      </div>
+      <div class="card-content">
+          <h2 class="project-name">${projectsList[i].name}</h2>
+          <ul class="triple">
+              <li class="canopy">${projectsList[i].company}</li>
+              <li class="double">${projectsList[i].role}</li>
+              <li class="double">${projectsList[i].year}</li>
+          </ul>
+          <p class="paragraph">
+              ${projectsList[i].description}
+          </p>
+          <ul class="langs">
+              <li class="lang1 languages">html</li>
+              <li class="lang2 languages">css</li>
+              <li class="lang3 languages">javascript</li>
+          </ul>
+          <button type="button" class="seeProject active project-modal-${i}">See Project</button>
+      </div>
+    </div>
+  `;
+}
+
+// POP UP WINDOW
+
+const popUp = document.createElement('div');
+popUp.classList.add('popUp');
+
+for (let i = 0; i < projectsList.length; i += 1) {
+  document.querySelector(`.project-modal-${i}`).addEventListener('click', () => {
+    popUp.style.display = 'block';
+    body.appendChild(popUp);
+    overlay.style.display = 'block';
+    popUp.innerHTML = `
+      <span><i class="fas fa-times close-btn"  id="closee"></i></span>
+      <div class="top-left">
+          <h2 class="popUp-title">${projectsList[i].name}</h2>
+          <ul class="popUp-list">
+              <li class="list-choice canopyy">${projectsList[i].company}</li>
+              <li class="list-choice">${projectsList[i].role}</li>
+              <li class="list-choice">${projectsList[i].year}</li>
+          </ul>
+          <div class="image-pop"><img src="./Images/Snapshoot-page-2.png" alt="" class="popUp-image"></div>
```

Comment:
- [ ] Your popup looks good but it has the same image repeating regardless of which project you click. Please implement the modal to show the project image corresponding to the project that was clicked 👍